clients/rospy/src/rospy/core.py

```diff
@@ -109,7 +110,7 @@ def parse_rosrpc_uri(uri):
     @raise ParameterInvalid: if uri is not a valid ROSRPC URI
     """
     if uri.startswith(ROSRPC):
-        dest_addr = uri[len(ROSRPC):]            
+        dest_addr = uri[len(ROSRPC):]

```

Comment:
Please take a look at the diff on the [GitHub PR](https://github.com/ros/ros_comm/pull/948/files) and update the PR to avoid any unnecessary changes.